autofit/graphical/optimise.py

```diff
@@ -51,19 +53,22 @@ def __init__(
             model_dist: Optional[MeanField] = None,
             transform: Optional[AbstractLinearTransform] = None,
             bounds: Optional[Dict[str, Tuple[float, float]]] = None,
-            method: str = 'L-BFGS-B',
+            method: str = 'L-BFGS-B', jac=False, tol=None, options=None, 
```

Comment:
I can't quite see where method gets used, but if you have a class that it retrieves then make the method that class and otherwise use an enum